src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Negotiate/NegotiateStream_Http_Tests.4.1.0.cs

```diff
@@ -96,6 +96,46 @@ public static void NegotiateStream_Http_AmbientCredentials()
         }
     }
 
+    [WcfFact]
+    [Condition(nameof(Windows_Authentication_Available),
+               nameof(Root_Certificate_Installed),
+               nameof(Ambient_Credentials_Available))]
+    [OuterLoop]
+    public static void NegotiateStream_Http_AmbientCredentialsForNet50()
+    {
+        string testString = "Hello";
+        ChannelFactory<IWcfService> factory = null;
+        IWcfService serviceProxy = null;
+
+        try
+        {
+            // *** SETUP *** \\
+            BasicHttpBinding binding = new BasicHttpBinding(BasicHttpSecurityMode.Transport);
+            binding.Security.Transport.ClientCredentialType = HttpClientCredentialType.Windows;
+            string host = TestProperties.GetProperty(TestProperties.ServiceUri_PropertyName);
+
+            factory = new ChannelFactory<IWcfService>(
+                binding,
+                new EndpointAddress(new Uri(Endpoints.Https_WindowsAuth_Address),new SpnEndpointIdentity($"HTTP/{host}")));
```

Comment:
Small nit, formatting. There should be a space after the comma. Can you check the formatting for all these changes?

Reply:
I have updated it